views/partials/fxa_enabled/fxa_navigation.hbs

```diff
@@ -0,0 +1,21 @@
+<nav id="global-navigation" class="global-navigation stack">
+  {{#if req.session.user }}
+    <div id="fxa-user-info" class="fxa-user-info">
+      <span id="user-email" class="user-email">{{ req.session.user.email }}</span>
+      <div class="avatar-wrapper">
+        <img class="avatar" src="{{ req.session.user.avatar }}"/>
+      </div>
+      <div class="fxa-menu">
+        <div class="fxa-links">
+          <a href="/">See Full Report</a>
+          <a href="https://accounts.firefox.com/">Manage Firefox Accounts</a>
+          <a href="/user/logout">Sign out</a>
+        </div>
+      </div>
+    </div> 
+  {{/if}}
+  {{#unless req.session.user }}
```

Comment:
Just curious: What's the advantage of doing an `{{#unless req.session.user }}` block instead of using `{{else}}` block in the `{{#if req.session.user }}`